### PR TITLE
validation changes

### PR DESCRIPTION
**Pull request by buwayogesh**

### CODE REVIEW

File: src/app/layout/default/default.component.html [hunk 1]
```diff
@@ -85,48 +85,48 @@ <h2 class="ftco-heading-2">Help Desk</h2>
...
+                  GPRA Quarters,Sector No. 26,
+                  Pradhikaran, Akurdi,
+                  <br>
+                  Pimpri-Chinchwad,
+                  Maharashtra 411044.
+                </span>
               </li>
               <li>
-                <a href="#"><span class="icon fa fa-phone"></span><span class="text">+2 392 3929 210</span></a>
+                <a class="ms-2" href="{{'tel:'+912047150760}}"><i class="mdi mdi-phone"></i><span class="text ms-2">   +91 20 47150760</span></a>
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.html:121]: static ahe phone no
<a class="ms-2" href="tel:+912047150760"><i class="mdi mdi-phone"></i><span class="text ms-2">   +91 20 47150760</span></a>

File: src/app/layout/default/default.component.ts [hunk 2]
```diff
@@ -40,14 +44,27 @@ export class DefaultComponent {
...
     } else {
       this.isLoggedin = true;
     }
-    this.loggedInUser = this.authService.getLoggedInUser();    
+    this.loggedInUser = this.authService.getLoggedInUser();
+    const requestOptions: Object = {
+      observe: 'body',
+      responseType: 'text',
+    };
+    this.http
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.ts:60]: move this to service

File: src/app/layout/default/default.component.ts [hunk 3]
```diff
@@ -124,4 +141,19 @@ export class DefaultComponent {
       });
     }
   }
+
+  // GetRssFeedData() {
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.ts:145]: remove commented code

File: src/app/modules/default.module.ts [hunk 4]
```diff
@@ -11,9 +11,12 @@ import { NgbDropdownModule, NgbPaginationModule, NgbTypeaheadModule } from '@ng-
 import { QuestionComponent } from '../pages/default/question/question.component';
 import { AllQuestionsComponent } from '../pages/default/all-questions/all-questions.component';
 import { TagsComponent } from '../pages/default/tags/tags.component';
+import { PrivacyPolicyComponent } from "../pages/privacy-policy/privacy-policy.component";
+import { RefundPolicyComponent } from "../pages/refund-policy/refund-policy.component";
+
 
 @NgModule({
-  declarations: [DefaultComponent, AllQuestionsComponent, AskQuestionComponent, QuestionComponent, TagsComponent],
+  declarations: [DefaultComponent, AllQuestionsComponent, AskQuestionComponent, QuestionComponent, TagsComponent, PrivacyPolicyComponent, RefundPolicyComponent],
```
Comment (shrikantpatwari) [inline src/app/modules/default.module.ts:19]: move to about module

File: src/app/pages/forgot-password/forgot-password.component.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
+@import './../../../assets/scss/mixins';
```
Comment (shrikantpatwari) [inline src/app/pages/forgot-password/forgot-password.component.scss:1]: Have you pulled latest changes yesterday ?

File: src/app/pages/forgot-password/forgot-password.component.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+.form-group {
```
Comment (shrikantpatwari) [inline src/app/pages/forgot-password/forgot-password.component.scss:2]: Use src/styles/_ftco-form for form style

**Review by shrikantpatwari — CHANGES_REQUESTED** [review]
Please make changes suggested

File: src/app/layout/default/default.component.html (hunk 1 above)
Comment (buwayogesh) [reply src/app/layout/default/default.component.html:121]: environment file madhe add karu ka?

File: src/app/pages/forgot-password/forgot-password.component.scss (hunk 5 above)
Comment (shrikantpatwari) [reply src/app/pages/forgot-password/forgot-password.component.scss:1]: check reset password component scss

File: src/app/layout/default/default.component.html (hunk 1 above)
Comment (shrikantpatwari) [reply src/app/layout/default/default.component.html:121]: Nako

File: src/styles/_ftco-form.scss [hunk 7]
```diff
@@ -1,3 +1,5 @@
+@import 'src/styles/mixins.scss';
```
Comment (shrikantpatwari) [inline src/styles/_ftco-form.scss:1]: etha kashala add kela he

File: src/app/routes/about.routes.ts [hunk 8]
```diff
@@ -1,12 +1,23 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AboutComponent } from '../pages/about/about.component';
+import { PrivacyPolicyComponent } from '../pages/privacy-policy/privacy-policy.component';
+import { RefundPolicyComponent } from '../pages/refund-policy/refund-policy.component';
 
 const routes: Routes = [
  {
     path:'',
     component: AboutComponent
```
Comment (shrikantpatwari) [inline src/app/routes/about.routes.ts:10]: working ahe ka ugich code commit kelay ?

File: src/app/pages/default/question/question.component.ts [hunk 9]
```diff
@@ -106,6 +106,13 @@ export class QuestionComponent implements OnInit {
   }
 
   submitAnswer(answer: {answerEditor: string}) {
-
+    if(this.questionId && answer.answerEditor.length) {
+      this.questionsService.patchAnswer(this.questionId,answer).subscribe((res:any) => {
```
Comment (shrikantpatwari) [inline src/app/pages/default/question/question.component.ts:110]: res: any ka ahe ?
kay problem ahe type denya sathi

File: src/app/layout/default/default.component.html [hunk 10]
```diff
@@ -138,9 +138,8 @@ <h2 class="ftco-heading-2">Have a Questions?</h2>
           <script>
             document.write(new Date().getFullYear());
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.html:139]: angular vapartoy na apan ka html javascript chalu kelay

File: src/app/layout/default/default.component.ts [hunk 11]
```diff
@@ -16,6 +16,10 @@ import { IResponse } from 'src/app/interfaces/response.interface';
 import { IQuestion } from 'src/app/interfaces/question.interface';
 import { Router } from '@angular/router';
 import { AuthService } from 'src/app/services/auth.service';
+import { HttpClient } from '@angular/common/http';
+import * as xml2js from 'xml2js';
+import { environment } from "src/environments/environment";
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.ts:21]: env and http client use nahi karat tar ka added ahe

File: src/app/layout/default/default.component.ts [hunk 12]
```diff
@@ -40,14 +46,22 @@ export class DefaultComponent {
   constructor(
     private questionService: QuestionsService,
     private authService: AuthService,
-    private router: Router
+    private router: Router,
+    private http: HttpClient
```
Comment (shrikantpatwari) [inline src/app/layout/default/default.component.ts:50]: same here

File: src/app/pages/default/question/question.component.ts (hunk 9 above)
Comment (shrikantpatwari) [inline src/app/pages/default/question/question.component.ts:110]: why any type? you don't know what is type of response ?

File: src/app/pages/profile/profile.component.ts [hunk 13]
```diff
@@ -154,8 +154,16 @@ export class ProfileComponent implements OnInit {
   }
 
   logout() {
-    localStorage.clear();
-    this.router.navigate(['/']);
+    this.authService.getUserLoggedOut().subscribe((res:any) => {
```
Comment (shrikantpatwari) [inline src/app/pages/profile/profile.component.ts:157]: type of res ?

File: src/app/pages/profile/profile.component.ts [hunk 14]
```diff
@@ -154,8 +154,16 @@ export class ProfileComponent implements OnInit {
...
+      if(res.status === 200) {
+        console.log(res);
```
Comment (shrikantpatwari) [inline src/app/pages/profile/profile.component.ts:159]: console log

**Review by shrikantpatwari — CHANGES_REQUESTED** [review]
Please check updated comments

**Pull request merged**